libpod/define/errors.go

```diff
@@ -12,15 +12,6 @@ var (
 	// ErrNoSuchPod indicates the requested pod does not exist
 	ErrNoSuchPod = errors.New("no such pod")
 
-	// ErrNoSuchImage indicates the requested image does not exist
-	ErrNoSuchImage = errors.New("no such image")
```

Comment:
This is a breaking change to the API, should we atlease set these constants to point at the libimage constants?

Reply:
Which API do you mean?

Reply:
`libimage` forwards all errors from `containers/storage`, so it would be `storage.ErrImageUnknown` but there's really no user of that error in libpod/Podman.

Reply:
If we change this then we should bump the version of podman/libpod to v3, if we are following the rules.